service/resource/certificate/desired.go

```diff
@@ -13,9 +13,9 @@ import (
 )
 
 const (
-	CaKey  = "ca"  // CaKey is the key in the Secret that holds the CA.
-	CrtKey = "crt" // CrtKey is the key in the Secret that holds the certificate.
-	KeyKey = "key" // KeyKey is the key in the Secret that holds the key.
+	caKey  = "ca"  // CaKey is the key in the Secret that holds the CA.
```

Comment:
This is me being defensive. We shouldn't expose this outside of the package (as they will be removed shortly).

Reply:
I think when you touched this code anyway and did a roundhouse kick anyway, you could have just use the certificatetpr constants.